public/index.html

```diff
@@ -2,21 +2,54 @@
+    <style>
+        .center_border {
+          margin: auto;
+          width: 60%;
+          border: 3px solid #006699;
+          padding: 10px;
+        }
+        .center_simple {
+          margin: auto;
+          width: 70%;
+        }
+        .center_text {
+          text-align: center
+        }
+    </style>
     <meta name="viewport" content="width=device-width, initial-scale=1">
 </head>
 <body>
 <header id="flex-header">
     <div class="cloud-header"></div>
     <h1>Congratulations!</h1>
-    <h2>You are currently running a Node.js app built for the IBM Cloud.</h2>
```

Comment:
@mdarden Since you changed this, you will have to change the corresponding functional test under `test/functional/test-routes.js`. It is looking for this specific text. That is why the checks for the PR aren't letting you merge.

Look for this code:
```
describe('GET /', () => {
  it('responds with homepage', () => {
    return request(server)
      .get('/')
      .expect('Content-Type', 'text/html; charset=UTF-8')
      .expect(200)
      .then(response => {
        expect(response.text).to.include(
          'You are currently running a Node.js app built for the IBM Cloud.',
        );
      });
  });
});
```